libs/features/mas/src/variants/ccd-slice.js

```diff
@@ -61,7 +61,7 @@ export class CCDSlice extends VariantLayout {
             line-height: var(--consonant-merch-card-body-xxs-line-height);
             min-width: 154px;
             max-width: 171px;
-            max-height: 54px;
+            max-height: 55px;
```

Comment:
is this meant to be here?

Reply:
yes, 54px does hide the second underline of links when focused. 
this doesn't have impact on the card height.

Reply:
got it, thx